### PR TITLE
mailmap: shiqi affiliation

### PR DESCRIPTION
**Pull request by ShiqiCooperation**
Signed-off-by: shiqi <1454927420@qq.com>

### CODE REVIEW

**Comment by dachary** [comment]
jenkins test this please

File: .organizationmap [hunk 1]
```diff
@@ -483,6 +483,7 @@ The Linux Box <contact@linuxbox.com> Casey Bodley <casey@linuxbox.com>
 The Linux Box <contact@linuxbox.com> Matt Benjamin <matt@linuxbox.com>
 The University of Arizona <contact@arizona.edu> James Ryan Cresawn <jrcresawn@gmail.com>
 Time Warner Cable Inc. <contact@twcable.com> Bryan Stillwell <bryan.stillwell@twcable.com>
+trendytech <contact@trendytech.com.cn> shiqi <m13913886148@gmail.com>
```
Comment (theanalyst) [inline .organizationmap]: maybe in caps the name of the company like `Trendy Tech` or so?

Comment (ShiqiCooperation) [inline .organizationmap]: OK, I change it to Trendy Tech

**Comment by badone** [comment]
Test this please

**Pull request merged**